GOVERNANCE.md

```diff
@@ -61,6 +61,35 @@ maintainer has already discussed the candidacy with the candidate and a
 maintainer is willing to be a sponsor by opening the pull request. The candidate
 becomes a maintainer once the pull request is merged.
 
+## Security Advisors
+
+A security advisor is an advisory role in the project responsible for helping
+classify and advise on embargoed security disclosures. Security advisors are
+individuals trusted by maintainers and representing significant users of the
+project.
+
+Security advisors are part of the organization without write access, but with
+read access to security disclosures and advisories before becoming public. There
+is no expectation of advisors to become reviewers or participate in issue
+triage and code review. Security advisors help maintain the integrity of the
```

Comment:
Should we add something here about developing threat models?

Reply:
Thread model development is a fairly specific task, I kind of left the role as more high level purpose. Maybe mentioning in the first sentence under "help classify" and "advise"